library/amazoncorretto

```diff
@@ -0,0 +1,5 @@
+Maintainers: Eric Edens (@ericedens)
```

Comment:
I'd strongly recommend listing a team email alias here (that can accept external mail) and/or a couple of other team members with GitHub accounts.

Reply:
Good idea! I've added the Corretto organization and you.